docs/sources/flow/getting-started/collect-opentelemetry-data.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Collect OpenTelemetry data
+weight: 300
+---
+
+# Collect OpenTelemetry data
+
+Grafana Agent Flow can be configured to collect [OpenTelemetry][] data and
+forward it to any OpenTelemetry-compatible database.
```

Comment:
nit: Technically it doesn't have to be a database. It could just be an "endpoint" which forwards it to another "endpoint"